hazelcast/src/main/java/com/hazelcast/client/impl/spi/impl/DefaultAddressProvider.java

```diff
@@ -48,17 +50,26 @@ public DefaultAddressProvider(ClientNetworkConfig networkConfig,
 
     @Override
     public Addresses loadAddresses() {
+        return null;
+    }
```

Comment:
Looks a bit dangerous and unexpected.

Why don't we leave this as it is, and just tweak the method below, using this?

Reply:
Removed this overload, as mentioned above.